compiler-rt/lib/builtins/fp_extend_impl.inc

```diff
@@ -37,71 +37,78 @@
 
 #include "fp_extend.h"
 
+#include <stdio.h>
```

Comment:
Is this include actually required or left over from debugging?

Reply:
good catch, no, it's a left over